JapaneseFilter/sections/specific.txt

```diff
@@ -2188,6 +2188,8 @@ navitime.co.jp###under-advertisement
 navitime.co.jp##.section-ad-1
 navitime.co.jp##.section-ad-3
 navitime.co.jp##.section-ad-4
+navitime.co.jp##.footer-rect-ad-frame
+navitime.co.jp###main-page-frame > div[style="display:flex;justify-content:center;align-items:center;height:120px;width:100%;"]
```

Comment:
`navitime.co.jp###main-page-frame > div[style="display:flex;justify-content:center;align-items:center;height:120px;width:100%;"]`

Not reproduced.

Reply:
URL: https://www.navitime.co.jp/transfer/pass/?from=top.topics

<details><summary>navitime.co.jp###main-page-frame > div[style="display:flex;justify-content:center;align-items:center;height:120px;width:100%;"]</summary>

![](https://i.imgur.com/FaOAUnA.png)
</details><br/>